src/tools/rbd_mirror/image_map/Policy.h

```diff
@@ -27,53 +27,53 @@ class Policy {
   }
 
   // init -- called during initialization
-  void init(
-      const std::map<std::string, cls::rbd::MirrorImageMap> &image_mapping);
+  void init(const std::map<GlobalId, cls::rbd::MirrorImageMap> &image_mapping);
 
   // lookup an image from the map
-  LookupInfo lookup(const std::string &global_image_id);
+  LookupInfo lookup(const GlobalId &global_id);
 
   // add, remove
-  bool add_image(const std::string &global_image_id);
-  bool remove_image(const std::string &global_image_id);
+  bool add_entity(const GlobalId &global_id, uint64_t weight);
+  bool remove_entity(const GlobalId &global_id);
 
   // shuffle images when instances are added/removed
-  void add_instances(const InstanceIds &instance_ids,
-                     GlobalImageIds* global_image_ids);
-  void remove_instances(const InstanceIds &instance_ids,
-                        GlobalImageIds* global_image_ids);
+  void add_instances(const InstanceIds &instance_ids, GlobalIds* global_ids);
```

Comment:
Nit:
```suggestion
  void add_instances(const InstanceIds &instance_ids, GlobalIds *global_ids);
```
and for other instances in this file.